src/factories/series/Column.ts

```diff
@@ -0,0 +1,92 @@
+module n3Charts.Factory.Series {
```

Comment:
Oh no no sorry, I was talking about `col` or `cols`, I think `column` and `columns` is not that longer and way more semantic.